mudlet-mapper.xml

```diff
@@ -4782,6 +4782,67 @@ function mmp.startup()
 end</script>
                 <eventHandlerList/>
             </Script>
+            <Script isActive="yes" isFolder="no">
+                <name>Remove other game's options on login</name>
+                <packageName></packageName>
+                <script>local whatGame = {
+belt={lusternia=true},
+blossom={lusternia=true},
+bonecurio={lusternia=true},
+caravan={imperian=true},
+cookie={lusternia=true},
+crowdmap={achaea=true},
+cubix={lusternia=true},
+dash={achaea=true,imperian=true,aetolia=true},
+duanathar={achaea=true},
+duanatharan={achaea=true},
+duantahar={achaea=true},
+duanatharic={achaea=true},
+facecurio={lusternia=true},
+feathercurio={lusternia=true},
+figurecurio={lusternia=true},
+fingerblade={lusternia=true},
+flowercurio={lusternia=true},
+fluttercurio={lusternia=true},
+head={lusternia=true},
+icicle={lusternia=true},
+lockwormholes={achaea=true},
+mandala={lusternia=true},
+medallion={lusternia=true},
+mud={lusternia=true},
+pebble={achaea=true},
+prism={lusternia=true},
+removewings={achaea=true,aetolia=true},
+runaway={achaea=true},
+screwdriver={lusternia=true},
+shackle={achaea=true, },
```

Comment:
looks like there is supposed to be another option here or there was and the comma didn't get removed